daemon/pkg/streamformatter/streamformatter.go

```diff
@@ -7,8 +7,8 @@ import (
 	"io"
 	"sync"
 
-	"github.com/docker/docker/pkg/jsonmessage"
-	"github.com/docker/docker/pkg/progress"
+	"github.com/docker/docker/daemon/pkg/jsonmessage"
+	"github.com/docker/docker/daemon/pkg/progress"
```

Comment:
`pkg/progress` and `pkg/streamformatter` are essential for the CLI; we need to look where to put those; possibly need to be split and/or moved as a whole to either `api/` or `client/`.

Reply:
Added to #50372